cmd/notation/inspect.go

```diff
@@ -235,17 +242,49 @@ func getSignedAttributes(outputFormat string, envContent *signature.EnvelopeCont
 	return signedAttributes
 }
 
-func getUnsignedAttributes(envContent *signature.EnvelopeContent) map[string]string {
-	unsignedAttributes := map[string]string{}
-
-	if envContent.SignerInfo.UnsignedAttributes.TimestampSignature != nil {
-		unsignedAttributes["timestampSignature"] = b64.StdEncoding.EncodeToString(envContent.SignerInfo.UnsignedAttributes.TimestampSignature)
-	}
+func getUnsignedAttributes(outputFormat string, envContent *signature.EnvelopeContent) map[string]any {
+	unsignedAttributes := make(map[string]any)
 
 	if envContent.SignerInfo.UnsignedAttributes.SigningAgent != "" {
 		unsignedAttributes["signingAgent"] = envContent.SignerInfo.UnsignedAttributes.SigningAgent
 	}
 
+	if envContent.SignerInfo.UnsignedAttributes.TimestampSignature != nil {
+		signedToken, err := tspclient.ParseSignedToken(envContent.SignerInfo.UnsignedAttributes.TimestampSignature)
+		if err != nil {
+			unsignedAttributes["timestampSignature"] = timestampOutput{
+				Error: "failed to parse timestamp countersignature",
+			}
+			return unsignedAttributes
+		}
+		info, err := signedToken.Info()
+		if err != nil {
+			unsignedAttributes["timestampSignature"] = timestampOutput{
+				Error: "failed to parse timestamp countersignature",
+			}
+			return unsignedAttributes
+		}
+		timestamp, err := info.Validate(envContent.SignerInfo.Signature)
+		if err != nil {
+			unsignedAttributes["timestampSignature"] = timestampOutput{
+				Error: "failed to parse timestamp countersignature",
+			}
+			return unsignedAttributes
+		}
+		certificates := getCertificates(outputFormat, signedToken.Certificates)
+		var formatTimestamp string
+		switch outputFormat {
+		case cmd.OutputJSON:
+			formatTimestamp = timestamp.Format(time.RFC3339)
+		default:
+			formatTimestamp = timestamp.Format(time.ANSIC)
+		}
+		unsignedAttributes["timestampSignature"] = timestampOutput{
+			Timestamp:             formatTimestamp,
+			TimestampCertificates: certificates,
+		}
+	}
+
 	return unsignedAttributes
```

Comment:
Simply curious about do we include `Certificate Chain` here? For the spec marked that as a REQUIRED attribute.